### PR TITLE
Refs #28077 -- Added opclasses to Index.__repr__().

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: django/db/models/indexes.py [hunk 1]
```diff
@@ -123,10 +123,11 @@ def set_name_with_model(self, model):
             self.name = 'D%s' % self.name[1:]
 
     def __repr__(self):
-        return "<%s: fields='%s'%s%s>" % (
+        return "<%s: fields='%s'%s%s%s>" % (
             self.__class__.__name__, ', '.join(self.fields),
             '' if self.condition is None else ', condition=%s' % self.condition,
             '' if not self.include else ", include='%s'" % ', '.join(self.include),
+            '' if not self.opclasses else ' opclasses=%s' % ', '.join(self.opclasses),
```
Comment (hannseman) [inline django/db/models/indexes.py:130]:
```suggestion
            '' if not self.opclasses else ", opclasses='%s'" % ', '.join(self.opclasses)
```

Comment (carltongibson) [reply django/db/models/indexes.py:130]: I don't think we want the comma between the `fields` and `opclasses` _attributes_ do we? 🤔

Looking at the test case, I'd imagine this: 

```
<Index: fields='headline, body' opclasses='varchar_pattern_ops, text_pattern_ops'>
```

which **does** add the quotes, to be consistent with `fields`...

```suggestion
            '' if not self.opclasses else " opclasses='%s'" % ', '.join(self.opclasses),
```

but it's not HTML, so I'm not 100% sure what's desired.

Comment (felixxm) [reply django/db/models/indexes.py:130]: I added quotes and the comma.

Comment (hannseman) [reply django/db/models/indexes.py:130]: I added the comma to be consistent with the `include` and `condition` attributes. Also, on `ExclusionConstraint` we separate attributes by comma but not on `UniqueConstraint`, so 🤷 indeed..

Comment (felixxm) [reply django/db/models/indexes.py]: Right, I removed commas between attributes (we don't have them in most of custom `__repr__()`\`s).

**Comment by felixxm** [comment]
@carltongibson @hannseman Thanks :+1:

**Pull request merged**